week-1-practise/level-3/1-piping.js

```diff
@@ -11,6 +11,7 @@
 */
 
 var startingValue = 5;
+const result = (startingValue * 2 + 10).toFixed(2);
```

Comment:
we want you to use the functions provided below (add, multiply, format) to achieve the same thing.

Reply:
let result = format(add((multiply(startingValue, 2)), 10));